### ISSUE TITLE
Audit use of term "owns", in "owned element" and "owning element"

### ISSUE THREAD

**Issue by spectranaut** [w3c/aria#1927]
This should haven after #1454 lands, as that removes the term "required owned element".

There are many uses of "owned elements" which should probably be reconsidered. For example, in [spinbutton](https://w3c.github.io/aria/#spinbutton):

> Authors MAY create a spinbutton with children or owned elements, but MUST limit those elements to a [textbox](https://w3c.github.io/aria/#textbox) and/or two [buttons](https://w3c.github.io/aria/#button).

Should we simply say "with accessibility children" or "with children" or (maybe) "with owned elements"?

Additionally, "owning element" is not used in the spec, except in [tooltip](https://w3c.github.io/aria/#tooltip) where it's meaning is not even the definition of "owning element":

> The tooltip typically becomes visible, after a short delay, in response to a mouse hover, or after the owning element receives keyboard focus

**Comment by spectranaut** [comment]
oops this is a duplicate of: https://github.com/w3c/aria/issues/1161
